crates/iota-indexer/src/models/transactions.rs

```diff
@@ -54,12 +61,69 @@ pub struct TxGlobalOrder {
     pub optimistic_sequence_number: Option<i64>,
 }
 
-impl From<&IndexedTransaction> for TxGlobalOrder {
+impl From<&IndexedTransaction> for CheckpointTxGlobalOrder {
     fn from(tx: &IndexedTransaction) -> Self {
-        TxGlobalOrder {
+        Self {
             global_sequence_number: tx.tx_sequence_number as i64,
             tx_digest: tx.tx_digest.into_inner().to_vec(),
-            optimistic_sequence_number: Some(0),
+            index_status: Some(IndexStatus::Started),
+        }
+    }
+}
+
+#[derive(Clone, Debug, Queryable, Insertable, QueryableByName, Selectable)]
+#[diesel(table_name = tx_global_order)]
+pub(crate) struct CheckpointTxGlobalOrder {
+    /// Number that represents the global ordering between optimistic and
```

Comment:
Maybe let's not repeat all the docstrings from the previous model, but just mention what are the differences, and why do we have a second model?

Now it's purpose is a bit confusing on the first glance, and the docstrings are more prone to getting outdated.